runtimes/common/src/lib.rs

```diff
@@ -196,3 +202,16 @@ where
 		})
 	}
 }
+
+pub struct DustAndFeeHandler<T>(sp_std::marker::PhantomData<T>);
```

Comment:
This sends dust to the treasury, right? Perhaps it could be reflected in the name of the struct. Something like `SendDustAndFeesToTreasury`.